python/mlc_chat/compiler/help.py

```diff
@@ -99,6 +99,9 @@
 (Experimental) The chunk size during prefilling. By default,
 the chunk size is the same as sliding window or max sequence length.
 This flag subjects to future refactoring.
+""".strip(),
+    """num_shards""": """
```

Comment:
let's later rename it to something more informative, such as `num_tp_shards`, or `num_tp_workers`. i'm not sure

Reply:
what about `tp_degree`?